### ISSUE TITLE
Not supported on 64bit version of iTunes

### ISSUE THREAD

**Issue by MenaceInc** [pezcode/ighp#2]
Recent update to iTunes made the whole program 64bit.
Tried creating a 64bit version using Visual Studio '13 but the resulting .dll wasn't recognised by iTunes.
iTunes gave a warning saying the plugin could not be run in 64bit mode and asked to contact the vendor. Cannot get it to bring the same warning up again, appears to be ignoring the plugin now.

**Comment by pezcode** [comment]
Thanks for the heads up, I haven't used iTunes in a while to be honest. I don't recall having trouble with 32 vs 64 bits back then. Was the main executable always 32-bit even in the 64-bit version and they just changed it now?
There's a new [Visualizer SDK](https://developer.apple.com/downloads/index.action?searchTextField=itunes visual plug-ins) released just last month that explicitly states 64-bit compatibility. I'll try it out and see if that fixes it. Kinda busy right now so this might take a week or two.

**Comment by MenaceInc** [comment]
Seems like it. It's also causing issues with the last.fm scrobbler. Don't worry about sorting it out quickly, seems that no one else has noticed judging by the lack of reports.

**Comment by pezcode** [comment]
Starting with iTunes 12.1 the 64-bit version now uses actual 64-bit binaries. What makes things more complicated is that there is now a compatibility version called "64-bit - for older video cards" which is still 32-bits but only installs on 64-bit Windows (https://discussions.apple.com/docs/DOC-6562).

The other problem was that parts of the SDK changed so that even the old 32-bit plugin doesn't work anymore. Not sure why they would do that without changing the plugin SDK versions but it was a simple fix.

Long story short, I just released version 0.1.1 along with a 64-bit binary and installer:
https://github.com/pezcode/ighp/releases
Give it a try and let me know if it works. I'll update the webpage later.

**Comment by MenaceInc** [comment]
Works perfectly so far, only tested setting hotkeys for play, next, previous and toggle random then using them from a different window but I don't foresee any issues. Thanks for updating the plugin, appreciate it.
